data/json/mapgen/mapgen-test.json

```diff
@@ -17,7 +17,7 @@
   {
     "id": "mapgen-test",
     "type": "overmap_terrain",
-    "name": "noname",
+    "name": "no name",
```

Comment:
I guess this is unneeded.

Reply:
This is to reduce the unwanted noise in CDDA corpus & word frequencies dataset I'm working on next.

Reply:
Ok, got it.

Reply:
By the way I just realised this should be moved to TEST DATA and should not be translated.